packages/vega-schema/src/axis.js

```diff
@@ -61,6 +61,7 @@ const axis = object({
   titleColor: colorValue,
   titleFont: stringValue,
   titleFontSize: numberValue,
+  titleFontStyle: stringValue,
```

Comment:
Since there are only three different possible values (https://developer.mozilla.org/en-US/docs/Web/CSS/font-style#Values), would it make sense to use an enum here?